src/styles/components/header.scss

```diff
@@ -1,2 +1,78 @@
 .header {
+    background: var(--color-background-darker);
+    font-size: 2rem;
+    font-weight: 600;
```

Comment:
lets make a variable for these weights, something like `--weight-semi-bold`

Reply:
Done